changelog.d/0-release-notes/get-mls-clients

```diff
@@ -0,0 +1 @@
+The `GET mls/clients` has been changed, and it now returns a pair of lists, instead of one
```

Comment:
Isn't this actually an internal endpoint, i.e., `GET /i/mls/clients`?

Reply:
Fixed